packages/ui-components/src/components/common/VideoWrapper.vue

```diff
@@ -0,0 +1,44 @@
+<template>
```

Comment:
The name is kinda weird
1. CommonVideoEmbed is more self explanatory
2. Better yet - CommonVimeoEmbed, since this doesn't handle any other kind of videos (and we might theoretically still want to embed YT videos in FE2 or other apps that might use ui-components)